### DISCUSSION TITLE
Kali in Windows 11 Problem

### DISCUSSION THREAD

**Discussion by Lopez017** [microsoft/WSL#10449] [General]
I tried to instal Kali Linux in my Windows 11, but I'm having a lot of issues HELP!!!!

Failed to attach disk 'C:\Users\Lopez\AppData\Local\Packages\KaliLinux.54290C8133FEE_ey8k8hqnwqnmg\LocalState\ext4.vhdx' to WSL2: The system cannot find the file specified.
Error code: Wsl/Service/CreateInstance/MountVhd/ERROR_FILE_NOT_FOUND

**Comment by str0mback** [comment]
That's because Windows Defender has quarantined several of the files.

My suggestion is to download Kali from their own repository, disable Defender, go offline and then install the image.

You will probably have to exclude \\wsl.localhost\Kali (or wherever you've got it located) from Defender, since it regards every tool Kali uses as a security risk (which, in the wrong hands, it is).
